packages/host/tests/acceptance/operator-mode-test.ts

```diff
@@ -1542,7 +1545,9 @@ module('Acceptance | operator mode tests', function (hooks) {
 
       assert.dom('[data-test-search-sheet]').hasClass('results'); // Search open
 
-      await waitFor(`[data-test-search-result="${testRealmURL}Pet/mango"]`);
+      await waitFor(`[data-test-search-result="${testRealmURL}Pet/mango"]`, {
+        timeout: 2000,
```

Comment:
I finally added this after it failed [four times](https://github.com/cardstack/boxel/actions/runs/6153655328/job/16698511404).